README.md

```diff
@@ -97,27 +100,49 @@ All the signatures used to execute the transaction are available at `tx.signatur
 Returns an instance of the Safe Core SDK with the `providerOrSigner` connected to the `safeAddress`.
 
 ```js
-const safeSdk = await EthersSafe.create(ethers, safeAddress, providerOrSigner)
+const safeSdk = await EthersSafe.create({ ethers, safeAddress, providerOrSigner })
```

Comment:
Making the params into an object is a breaking change, right? 

This is not a problem (as we are at a 0.x version), just wondering ;)